### ISSUE TITLE
Non-black transparent colour is interpreted as solid colour in 4bpp

### ISSUE THREAD

**Issue by mattuna15** [yeastplume/aloevera#10]
I mistakenly added transparent white instead of transparent black to palette. Aseprite sees it as transparent. Passed all validation rules but on load in game - showed as solid white (correctly). I believe only transparent black (00000) is seen as transparent. Is it possible to add warning when formatting imageset in 4bpp that a non-black transparent colour has been selected?

**Comment by yeastplume** [comment]
Right, Aloevera (and VERA) don't have any concept of alpha, but I see how that can cause some confusion and is definitely worth a warning. I'll insert this along with a few of the recent fixes for a 0.2.1 release at some stage next week.

**Comment by mattuna15** [comment]
Yes - threw me for a minute before I realised what I'd done!
